spec/gilded_rose_spec.js

```diff
@@ -1,7 +1,206 @@
+    it("Then normal item decreases twice in quality", function() {
+      items[0] = new Item('normal item', -1, 5)
+      update_quality();
+      expect(items[0].sell_in).toEqual(-2)
+      expect(items[0].quality).toEqual(3)
+    })
+    it("Then normal item quality doesn't go < 0", function() {
+      items[0] = new Item('normal item', -1, 1)
+      update_quality();
+      expect(items[0].sell_in).toEqual(-2)
+      expect(items[0].quality).toEqual(0)
+    })
+  })
+
+  describe("When sell in <0", function() {
+    it("Then normal item decreases to zero when quality <2", function() {
+      items[0] = new Item('normal item', -1, 1)
+      update_quality();
+      expect(items[0].sell_in).toEqual(-2) //-2
+      expect(items[0].quality).toEqual(0) //0
+    })
+  })
+
+  describe("When sell in = 0", function() {
+    it("Then normal item quality decreases by -2", function() {
+      items[0] = new Item('normal item', 0, 6)
+      update_quality();
+      expect(items[0].sell_in).toEqual(-1) // -1
+      expect(items[0].quality).toEqual(4) //4
+    })
+  })
+
+  describe("When quality reaches 0", function() {
+    it("cannot decrease anymore", function() {
+      items[0] = new Item('cat', 0, 0)
+      update_quality();
+      expect(items[0].sell_in).toEqual(-1)
+      expect(items[0].quality).toEqual(0)
+    })
+  })
+
+  describe("When quality reaches 0", function() {
+    it("cannot decrease anymore", function() {
+      items[0] = new Item('cat', 0, 1)
+      update_quality();
+      expect(items[0].sell_in).toEqual(-1)
+      expect(items[0].quality).toEqual(0)
+    })
+  })
+
+  describe("When Aged Brie is the item", function() {
+    it("quality increases", function() {
+      items[0] = new Item('Aged Brie', 5, 10)
+      update_quality();
+      expect(items[0].sell_in).toEqual(4) //4
+      expect(items[0].quality).toEqual(11) //11
+    })
+  })
+  // describe("When Aged Brie is the item and quality < 0", function() {
+  //   it("quality increases normally", function() {
+  //     items[0] = new Item('Aged Brie', -2, -3)
+  //     update_quality();
+  //     expect(items[0].sell_in).toEqual(0) //-3
+  //     expect(items[0].quality).toEqual(0) //1
+  //   })
+  // })
+  describe("When Aged Brie sell_in < 0", function() {
+    it("quality increases by 2", function() {
+      items[0] = new Item('Aged Brie', -1, 10)
+      update_quality();
+      expect(items[0].sell_in).toEqual(-2) //-2
+      expect(items[0].quality).toEqual(12) //12
+    })
+  })
+  describe("When Backstage pass sell_in < 0", function() {
+    it("quality remains 0", function() {
+      items[0] = new Item('Backstage passes to a TAFKAL80ETC concert', -1, 0)
+      update_quality();
+      expect(items[0].sell_in).toEqual(-2) //-2
+      expect(items[0].quality).toEqual(0) //0
+    })
+  })
+
+  describe("When quality reaches 50", function() {
+    it("cannot increase anymore", function() {
+      items[0] = new Item('Aged Brie', 12, 50)
```

Comment:
Too may items under tests. Multiple `expects` can be a sign of test that is too broad or not well defind.